solidity/contracts/price-feed/BTCETHPriceFeed.sol

```diff
@@ -11,9 +11,10 @@ contract BTCETHPriceFeed is Ownable, IBTCETHPriceFeed {
     using SafeMath for uint256;
 
     bool private _initialized = false;
+    address internal tbtcSystemAddress;
 
-    IMedianizer btcPriceFeed;
-    IMedianizer ethPriceFeed;
+    IMedianizer[] private btcUsdFeeds;
+    IMedianizer[] private ethUsdFeeds;
```

Comment:
Really it's two changes here… We'll have a single direct BTCETH price feed, in addition to this way of dealing with failures.

Reply:
Yep, we just need one array